torch/_dynamo/guards.py

```diff
@@ -127,6 +127,7 @@ def uninteresting_files():
             if isinstance(a, (np.generic, np.ndarray))
             else a,
         ),
+        ("torch", torch),
```

Comment:
currently we have a global `device` set to be `torch.device`. I wonder whether just exposing torch is alright. I'll  let the others reveiw this one